### ISSUE TITLE
Free Kill Slayer is not working

### ISSUE THREAD

**Issue by superblack4** [azalty/sm-hosties-plus#14]
Hello, 

Me and my friends test on JB server with this plugin. However, i have set sm_hosties_freekill_treshold "1" and when I kill a second non rebel prisoner, i won't be slay.

**Comment by azalty** [comment]
Thanks for reporting, I'll investigate, since I don't personally use this feature.

It seems like the convar's description is wrong as well, it is NOT `The amount of non-rebelling terrorists a CT is allowed to kill before action is taken` but rather `The amount of non-rebelling terrorists killed by a CT after which action is taken`, so `sm_hosties_freekill_treshold "1"` should punish the CT for every kill on a non-rebelling terrorist.

If you have any other info, feel free to share :)

**Comment by superblack4** [comment]
Yeah but if i kill more than 1 then i suppose to be slay. In my case, it
does not slay CT side

**Comment by azalty** [comment]
> Yeah but if i kill more than 1 then i suppose to be slay. In my case, it does not slay CT side

Yea I understood that. I'll investigate this bug.

Watch out by the way, your email replies leak some info, you might want to comment through the website. I've edited your comment to remove some info.

**Comment by superblack4** [comment]
1 more thing, when CT freekill a prisoner, it does not play the freekill
sound. I have put sound both in my server and FASTDL too

**Comment by azalty** [comment]
The issue might be that freekills are just not detected at all

Watch out by the way, your email replies leak some info, you might want to comment through the website. I've edited your comment to remove some info.

**Comment by superblack4** [comment]
thank you for noticing me
